kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -109,15 +110,15 @@ class CategoryDataType(Enum):
 
 class BlueBrainNexus(Store):
```

Comment:
this has nothing to do with the paths  ... u.u

Reply:
No it doesn't, the IDE was complaining that the signature of the implementation was not the same as the superclass so I made them match (just added a = None) and then I reformatted while I was there